.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -3,3 +3,6 @@ contact_links:
   - name: Account Limit Increase Request
     url: https://mailxto.com/xy5ijb
     about: Submit a request to increase the size limit on your Web3.Storage account!
+  - name: Pinning Service Access Request
```

Comment:
Hi @orvn!
This is already in `main` (see [config.yml](https://github.com/web3-storage/web3.storage/blob/main/.github/ISSUE_TEMPLATE/config.yml)).
There's also copy in the [`mailto`](https://mailxto.com/ta3m8b) link  from @dchoi27.

Reply:
👋 @LeslieOA! This was actually a product of running `git merge main`. Because this and `feat/new-brand-frontend` are both WIP branches. So sometimes you'll see things that are already in `main`, when PR'd against the destination `feat/new-brand-frontend`. 

Since the commits are being preserved before the squash + merge, it shouldn't result in any merge conflicts when this is fully ready to ship out!

Reply:
@orvn Ah! Apologies; I should have checked your target branch.

Reply:
No worries, tbh I wish the Github UI would make PRs to a non-standard destination branch stand out more—they're easy to overlook